GeneratorInterface/GenFilters/interface/LHEGenericFilter.h

```diff
@@ -49,8 +49,10 @@ class LHEGenericFilter : public edm::EDFilter {
   
   edm::EDGetTokenT<LHEEventProduct> src_;
   int numRequired_;                // number of particles required to pass filter
-  bool acceptMore_;                // if true (default), accept numRequired or more.
-                                   // if false, accept events with exactly equal to numRequired.
+  std::string acceptLogic_;        // LT  meaning <
```

Comment:
could you store this as an enum both to ensure that its always valid and to make the string comparison only once rather than many times?
@perrozzi